README.md

```diff
@@ -1 +1,79 @@
-# school-library-oop
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School library
+
+> This app is a tool one can use in a library to record what books are in the library and who borrows them. This app will allow a user to:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a book and when.
+
+## Screenshots
+
+> |Screenshot 1|Screenshots 2|
+> |--------------|----------------|
+> |![](app1.png)|![](app2.png)|!
+> |Screenshot 3|Screenshots 4|
+> |--------------|----------------|
+> |![](app3.png)|![](app4.png)|!
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+Ruby
+```
+
+### Setup
+```
+git clone git@github.com:Tchilo/school-library.git
+cd school-library.git
+```
+
+### Run Program
+```
+ruby school-library.rb
```

Comment:
- [ ] Your app has no entry point yet and there is really no `school-library` file in your app, so I'd suggest you remove this or use some other file that exists (as some kind of placeholder).

Reply:
Thanks I will change that. 😄